compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -1072,8 +1079,8 @@ OMR::CodeCacheManager::carveCodeCacheSpaceFromRepository(size_t segmentSize,
    if (config.verboseCodeCache())
       {
       if (start)
-         TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "carved size=%u range: " POINTER_PRINTF_FORMAT "-" POINTER_PRINTF_FORMAT,
-                  codeCacheSizeToAllocate, start, end);
+         TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "carved size=%u isAvailableMemorySafeForCarving=%d range: " POINTER_PRINTF_FORMAT "-" POINTER_PRINTF_FORMAT,
```

Comment:
This information is more important when we cannot do the allocation.

Reply:
Moved the extra verbose to the failure case.